doc/playbook.rst

```diff
@@ -483,9 +515,30 @@ From ``rubin-devl``, new APDB schemas can be created in the usual way:
    apdb-cli create-sql --namespace="pp_apdb_lsstcomcamsim" \
        "postgresql://rubin@usdf-prompt-processing-dev.slac.stanford.edu/lsst-devl" apdb_config_lsstcomcamsim.py
 
+Cassandra
+^^^^^^^^^
+
+To set up a new keyspace and connection, use:
+
+.. code-block:: sh
+
+   apdb-cli create-cassandra sdfk8sk001.sdf.slac.stanford.edu sdfk8sk004.sdf.slac.stanford.edu \
+       pp_apdb_latiss pp_apdb_latiss-dev.py --replication-factor=3 --enable-replica
+   apdb-cli create-cassandra sdfk8sk001.sdf.slac.stanford.edu sdfk8sk004.sdf.slac.stanford.edu \
+       pp_apdb_hsc pp_apdb_hsc-dev.py --replication-factor=3 --enable-replica
+   apdb-cli create-cassandra sdfk8sk001.sdf.slac.stanford.edu sdfk8sk004.sdf.slac.stanford.edu \
+       pp_apdb_lsstcomcamsim pp_apdb_lsstcomcamsim-dev.py --replication-factor=3 --enable-replica
+
+Here ``sdfk8sk001.sdf.slac.stanford.edu`` and ``sdfk8sk004.sdf.slac.stanford.edu`` are two nodes within the Prompt Processing allocation, which are the ``contact_points`` used for the initial connection.
+All of the available nodes will be used.
+In the above example, ``pp_apdb_latiss`` is the Cassandra keyspace (similar to schema for Postgres), and ``pp_apdb_latiss-dev.py`` is the usual APDB config.
+
 Resetting the APDB
 ------------------
 
+Postgres
+^^^^^^^^
+
 To restore the APDB to a clean state, run ``apdb-cli create-sql`` with ``--drop`` option which recreates all tables:
```

Comment:
I'm not sure splitting up this section makes sense -- it's literally just the one argument.

Reply:
I agree, but since we have code examples it seemed more clear to just include both. I could remove the subsections and change the text to:

> To restore the APDB to a clean state, add the ``--drop`` option to  ``apdb-cli create-sql`` or ``apdb-cli create-cassandra``  which will recreate all tables: